server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMessagesMethodTest.java

```diff
@@ -3204,7 +3204,6 @@ public void mailboxIdsShouldBeInDestinationWhenUsingForMove() throws Exception {
 
     @Test
     public void mailboxIdsShouldBeInDestinationWhenUsingForMoveWithoutTrashFolder() throws Exception {
-        mailboxProbe.deleteMailbox(MailboxConstants.USER_NAMESPACE, USERNAME, DefaultMailboxes.TRASH);
```

Comment:
you are not matching the test anymore?

Reply:
Euh... The test don't make sens as we auto-provision trash folder.

BTW that folder was created in the setUp. As I remove it's creation in the setUp so do I here!

Reply:
if you remove it after the auth, you should be able to really remove it (or maybe it's forbidden?).
But as it should never been absent, maybe you can remove this test

Reply:
ok